### DISCUSSION TITLE
.env RPC_URL and PRIVATE_KEY having issue in it

### DISCUSSION THREAD

**Discussion by Hasan2k5** [Cyfrin/foundry-full-course-cu#3049] [General]
I have stored the **PRIVATE_KEY** and **RPC_URL** in the **.env** file. However, when I try to use the private key and RPC URL through the following command, I get an error.

![image](https://github.com/user-attachments/assets/6870a0e8-a33d-4322-9f63-d53e108a4612)

here is also a SS of the .env file 
![image](https://github.com/user-attachments/assets/6cc54932-2687-4ec2-9b5a-dfd5db4e7ff6)

**Comment by EngrPips** [comment]
Hello @Hasan2k5,  First of all, I would like to address the fact that you show your private key publicly like this. It is unacceptable behavior, so please don't ever do it again. Yeah, I know it is an Anvil private key, but if you get into the habit of showing it like this, you will do the same for a real private key.

So there should be no space between `PRIVATE_KEY`, `=`, and `0x59c6...`, and it should work.
